tools/accuracy_checker/accuracy_checker/metrics/regression.py

```diff
@@ -542,11 +542,15 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
+    annotation_val = annotation_val.astype(float) if annotation_val.dtype == np.uint8 else annotation_val
```

Comment:
I think it should be more generic (if value is integer then cast to float)
np.issubdtype(annotation_value.dtype, np.integer)

Reply:
and probably before diff call in base class to prevent the issue for other regression metrics)

Reply:
Fixed